packages/www/components/Dashboard/StatusBadge/index.tsx

```diff
@@ -51,7 +53,11 @@ const StatusBadge = ({
   if (!timeAgo.isValid() || style.noTooltip) {
     return badge;
   }
-  return <Tooltip content={timeAgo.fromNow()}>{badge}</Tooltip>;
+  let contentText = timeAgo.fromNow();
+  if (tooltipText) {
+    contentText = `${tooltipText} ${contentText}`;
+  }
+  return <Tooltip content={contentText}>{badge}</Tooltip>;
```

Comment:
**nit:** Just naming, wdyt of `tootlipPrefix` for the arg received externally, and then we use that to create `tooltipText`?
```suggestion
  let tooltipText = timeAgo.fromNow();
  if (tooltipPrefix) {
    tooltipText = `${tooltipPrefix} ${tooltipTexst}`;
  }
  return <Tooltip content={tooltipText}>{badge}</Tooltip>;
```

Reply:
It does make sense, but if I update the logic allowing `tooltipText` to be also without a timestamp for different kind of status badges, it's not necessarily a prefix, wdyt?

Reply:
What if it's a function that receives the timeAgo (if a timestamp was provided) and returns the final tooltip text? Or maybe it should just be a string and we can repeat the timeAgo logic on the webhooks table anyway. Could create a helper if it feels cumbersome to repeat, but I think it should be ok either way (callback might be overengineering and suffer from React callback hell)